lib/parser/objects.ts

```diff
@@ -30,7 +27,10 @@ export class OI implements Position {
     if (this.i_ === undefined) {
       this.calcI();
     }
-    return this.i_ as number;
+    if (this.i_ === undefined) {
+      throw new Error('Can not convert position to i');
+    }
```

Comment:
maybe move the check in the if condition above. Otherwise this looks a bit weird why it is checked twice...